src/facets/BoostFacet.sol

```diff
@@ -18,13 +18,15 @@ error BoostFacet__InvalidFeeReceivers();
 /// @notice Facet in charge of point's boosts
 /// @dev Utilizes 'LDiamond', 'AppStorage'
 contract BoostFacet {
+
+    /// @notice Ordering of the events are according to their relevance in the facet
     event ClaimBoost(
-        address indexed _user,
         uint256 _seasonId,
```

Comment:
Should we prefer to use `indexed` for seasonId?

Reply:
We can still catch them inside subgraph and update entities.. @mejiasd3v  @Thehitesh172 thoughts?

Reply:
yes, we should always index user and seasonId